fulp_modules/bloodsuckers/bloodsucker/bloodsucker.dm

```diff
@@ -0,0 +1,746 @@
+	to_chat(owner.current, "<span class='notice'>Your existing powers have all ranked up as well!</span>")
+	update_hud(owner.current)
+	owner.current.playsound_local(null, 'sound/effects/pope_entry.ogg', 25, TRUE, pressure_affected = FALSE)
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+//Assign default team and creates one for one of a kind team antagonists
+
+// Create Objectives
+
+/datum/antagonist/bloodsucker/proc/forge_bloodsucker_objectives() // Fledgling vampires can have different objectives.
+
+	// TEAM
+	//clan = new /datum/team/vampireclan(owner)
+
+	// Lair Objective:		Create a Lair
+	var/datum/objective/bloodsucker/lair/lair_objective = new
+	lair_objective.owner = owner
+	lair_objective.generate_objective()
+	add_objective(lair_objective)
+
+	// Protege Objective
+	var/datum/objective/bloodsucker/protege/protege_objective = new
+	protege_objective.owner = owner
+	protege_objective.generate_objective()
+	add_objective(protege_objective)
+
+	//if (rand(0,1) == 0)
+		// Heart Thief Objective
+	var/datum/objective/bloodsucker/heartthief/heartthief_objective = new
+	heartthief_objective.owner = owner
+	heartthief_objective.generate_objective()
+	add_objective(heartthief_objective)
+	/*
+	else
+		// Solars Objective, doesnt work due to TG updates.
+		var/datum/objective/bloodsucker/solars/solars_objective = new
+		solars_objective.owner = owner
+		solars_objective.generate_objective()
+		add_objective(solars_objective)
+*/
+	// Survive Objective
+	var/datum/objective/bloodsucker/survive/survive_objective = new
+	survive_objective.owner = owner
+	survive_objective.generate_objective()
+	add_objective(survive_objective)
+
+/datum/antagonist/bloodsucker/proc/add_objective(var/datum/objective/O)
+	O.team = src
+	O.update_explanation_text()
+	objectives += O
+
+/datum/antagonist/bloodsucker/proc/remove_objectives()
+	var/datum/team/team = get_team()
+	if(team)
+		team.remove_member(owner)
+
+	for(var/O in objectives)
+		objectives -= O
+		qdel(O)
+
+//Name shown on antag list
+/datum/antagonist/bloodsucker/antag_listing_name()
+	return ..() + "([ReturnFullName(TRUE)])"
+
+//Whatever interesting things happened to the antag admins should know about
+//Include additional information about antag in this part
+/datum/antagonist/bloodsucker/antag_listing_status()
+	if (owner && owner.AmFinalDeath())
+		return "<font color=red>Final Death</font>"
+	return ..()
+
+
+//  		   Breakdown of Bloodsuckers:
+
+//					G A M E P L A Y
+//
+//	Bloodsuckers should be inherrently powerful: they never stay dead, and they can hide in plain sight
+//  better than any other antagonist aboard the station.
+//
+//	However, only elder Bloodsuckers are the powerful creatures of legend. Ranking up as a Bloodsucker
```

Comment:
fulp: let's remove heretics from active rotation, since they're an antag that's balanced around them being able to be validhunted down when they're weak before they can power up
also fulp: let's add a new antag that starts weak and then becomes strong as they power up over the course of the round
🤔

Reply:
I kid, I kid. Still, it makes ya think.

Reply:
Heretics is being put in rotation soon, and even then, Bloodsuckers at max level doesnt compare to an ascended heretic.